src/python/WMCore/WMSpec/StdSpecs/PromptReco.py

```diff
@@ -19,69 +19,6 @@
 from WMCore.WMSpec.StdSpecs.StdBase import StdBase
 from WMCore.WMSpec.WMWorkloadTools import makeList, strToBool
 
-def injectIntoConfigCache(frameworkVersion, scramArch, initCommand,
-                          configUrl, configLabel, couchUrl, couchDBName,
-                          envPath = None, binPath = None):
-    """
-    _injectIntoConfigCache_
-    """
-    logging.info("Injecting to config cache.\n")
-    configTempDir = tempfile.mkdtemp()
-    configPath = os.path.join(configTempDir, "cmsswConfig.py")
-    configString = urllib.urlopen(fixCVSUrl(configUrl)).read(-1)
-    configFile = open(configPath, "w")
-    configFile.write(configString)
-    configFile.close()
-
-    scramTempDir = tempfile.mkdtemp()
-    wmcoreBase = getWMBASE()
-    if not envPath:
-        envPath = os.path.normpath(os.path.join(wmcoreBase, "../../../../../../../../apps/wmagent/etc/profile.d/init.sh"))
-    scram = Scram(version = frameworkVersion, architecture = scramArch,
-                  directory = scramTempDir, initialise = initCommand,
-                  envCmd = "source %s" % envPath)
-    scram.project()
```

Comment:
Actually, re: the e-mail thread, I see at least 4 imports pylint is complaining about are no longer needed when you remove this block of code.